profiles.yml

```diff
@@ -6,13 +6,27 @@ jaffle_shop:
   outputs:
     dev:
       type: bigquery
-      method: service-account # for production runs: https://docs.getdbt.com/docs/profile-bigquery#section-service-account-file-authentication
-      project: wam-bam-258119
-      dataset: dbt_bq_example # You can also use "schema" here
+      method: oauth # for production runs: https://docs.getdbt.com/docs/profile-bigquery#section-service-account-file-authentication
```

Comment:
make the default oauth and allow the developer to use the commented out service account config as desired